frontend/src/theme/selectStyle.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from "./colors";
+
+const optionBackgroundColor = (isSelected: boolean, isFocused: boolean) => {
+  if (isSelected) {
+    return colors.teal[100];
+  }
+  if (isFocused) {
+    return colors.teal[50];
+  }
+  return colors.white.default;
+};
+
+const selectStyle: any = {
+  singleValue: (provided: any) => ({
+    ...provided,
+    color: colors.gray[500],
```

Comment:
the default color text should be teal.300
```suggestion
    color: colors.teal[300],
```

I don't know where to make the same change so that the tags select uses teal.300 😅 

Also, when the select is focused, the color should update to teal.400. I think you have to pass state as a parameter, then do something like:
``` suggestion
    color: state.isFocused ? colors.teal[400] : colors.teal[300],
```